engine/src/flutter/impeller/display_list/canvas.cc

```diff
@@ -540,6 +540,26 @@ void Canvas::DrawRoundRect(const RoundRect& round_rect, const Paint& paint) {
   DrawPath(path, paint);
 }
 
+void Canvas::DrawRoundSuperellipse(const RoundSuperellipse& rse,
+                                   const Paint& paint) {
+  if (paint.style == Paint::Style::kFill) {
+    Entity entity;
```

Comment:
Another future work to track. If the style is fill and the mask radius is larger than a threshold then we could approximate the shadow with the fast_rrect_blur - but we'd have to be careful that there isn't a jump when we exceed that threshold...

Reply:
Done https://github.com/flutter/flutter/issues/163893